backend/src/auth/auth.guard.ts

```diff
@@ -26,9 +28,19 @@ export class AuthGuard implements CanActivate {
       const payload = await this.jwtService.verifyAsync<JwtPayload>(token, {
         secret: this.config.getOrThrow('SECRET'),
       });
-      // 💡 We're assigning the payload to the request object here
-      // so that we can access it in our route handlers
-      request['user'] = payload;
+
+      const user = await this.usersService.findOneById(payload.sub);
+
+      if (!user) {
+        throw new UnauthorizedException();
+      }
+
+      console.log(JSON.stringify(user));
+      console.log(
+        'AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
```

Comment:
Ist das Kunst oder kann das weg?
```suggestion
      console.log(JSON.stringify(user));
      console.log(
        'AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
```